### PR TITLE
Adkernel adapter

### PR DESCRIPTION
**Pull request by ckbo3hrk**

### CODE REVIEW

**Comment by mansinahar** [comment]
Hey @ckbo3hrk! Wondering how this would be different from the [AdkernelAdn](https://github.com/prebid/prebid-server/tree/master/adapters/adkernelAdn) we already have?

**Comment by ckbo3hrk** [comment]
> Hey @ckbo3hrk! Wondering how this would be different from the [AdkernelAdn](https://github.com/prebid/prebid-server/tree/master/adapters/adkernelAdn) we already have?

@mansinahar Despite their similarity these two adapters are designed to work with two different platforms and each adapter differs in terms of config parameters, endpoints and imp.banner initialization. There is the same situation with adkernel & adkernelAdn adapters in prebid.js.

**Comment by mansinahar** [comment]
@ckbo3hrk I, see. Thanks! Might be good to add a doc in https://github.com/prebid/prebid-server/tree/master/docs/bidders explaining the key differences for Publishers to consider when choosing from either or. Or even both, for that matter.

File: adapters/adkernel/adkernel.go [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+		bidRequest.App.Publisher = nil
+	}
+	return &bidRequest
+}
+
+// Builds endpoint url based on adapter-specific pub settings from imp.ext
+func (adapter *adkernelAdapter) buildEndpointURL(params *openrtb_ext.ExtImpAdkernel) (string, error) {
+	reqHost := params.Host
+	zoneIdStr := strconv.Itoa(params.ZoneId)
+	endpointParams := macros.EndpointTemplateParams{Host: reqHost, ZoneID: zoneIdStr}
```
[the remark this answers is not in the document]
Comment (ckbo3hrk) [reply adapters/adkernel/adkernel.go]: @mansinahar Thank you for the review. Fixed in https://github.com/prebid/prebid-server/pull/997/commits/8c289bf70ef524da7ebb6101c97b2a66aef14bb1

**Pull request merged**